include/dxc/DXIL/DxilTypeSystem.h

```diff
@@ -95,7 +94,7 @@ class DxilFieldAnnotation {
   bool m_bCBufferVarUsed; // true if this field represents a top level variable in CB structure, and it is used.
 };
 
-class DxilTemplateArgAnnotation : DxilFieldAnnotation {
+class DxilTemplateArgAnnotation {
```

Comment:
There wasn't any good reason this was privately derived from DxilFieldAnnotation.  It simply bloated the class.